### ISSUE TITLE
Add `yAxis.lineDashStyle` to control the yAxis vertical line style

### ISSUE THREAD

**Issue by ItaiYosephi** [highcharts/highcharts#21209]
# Description of the feature
add the ability to control the dash style of teh vertical yAxis line

# Proof of Concept/Live example for the feature
Ability to make the red line in the example dashed:
https://jsfiddle.net/9hxrot1y/5/

**Comment by karolkolodziej** [comment]
Hi @ItaiYosephi and thank you for posting this idea.

In the meantime, you can use the workaround from [the demo](https://jsfiddle.net/BlackLabel/2c9Ljz8o/) and manually change the `dashStyle` of the axis line.

```js
        this.yAxis[0].axisLine.attr({
          dashstyle: 'ShortDash'
        })
```

**Comment by ItaiYosephi** [comment]
> Hi @ItaiYosephi and thank you for posting this idea.
> 
> In the meantime, you can use the workaround from [the demo](https://jsfiddle.net/BlackLabel/2c9Ljz8o/) and manually change the `dashStyle` of the axis line.
> 
> ```js
>         this.yAxis[0].axisLine.attr({
>           dashstyle: 'ShortDash'
>         })
> ```
Hi @karolkolodziej 
Thanks for your help! it does work, but I wonder why typescript does not recognize `axisLine` exits on `Axis`

**Comment by karolkolodziej** [comment]
You are right the `axisLine` isn't publically available but you can add it so TS understands it exists- [see documentation. ](https://www.highcharts.com/docs/advanced-chart-features/highcharts-typescript-declarations#extending-highcharts-typing)
Or you can check directly in [the demo.](https://stackblitz.com/edit/highcharts-angular-line-pm2xws?file=src%2Fapp%2Fapp.component.ts)
